packages/flutter/lib/src/material/dialog.dart

```diff
@@ -541,6 +541,7 @@ class SimpleDialogOption extends StatelessWidget {
     Key key,
     this.onPressed,
     this.child,
+    this.padding,
```

Comment:
nit: put this parameter above child, since there's a preference for `child` parameters to be the last param in a named parameter list.